config/routes.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
-  root "users#new"
```

Comment:
kept `users#new` so that the existing feature spec will still pass, but if we are going to continue on this path I will remove it